packages/portal/src/index.tsx

```diff
@@ -64,6 +69,12 @@ type PortalProps = {
    * @see Docs https://reach.tech/portal#portal-type
    */
   type?: string;
+  /**
+   * Target DOM element to render the portal.
+   *
+   * @see Docs https://reach.tech/portal#portal-nodeContainer
+   */
+  nodeContainer?: HTMLElement;
```

Comment:
Let's make `container` of type `Node`, `Document` is the default and really is can be any Node type.